App.tsx

```diff
@@ -25,11 +26,14 @@ const Navigator = createStackNavigator(
   }
 );
 
-const store = createStore();
+const { store, persistor } = createStore();
 const App: React.StatelessComponent<{}> = () => {
   return (
     <Provider store={store}>
-      <Navigator />
+      {/* TODO: decide if we should show a loading indicator here. */}
+      <PersistGate loading={null} persistor={persistor}>
```

Comment:
Is it standard for this to be null? I'd imagine this to be a boolean or optional.

Reply:
That's from the docs and the typings don't seem to be complaining, so guess that's how this library does it lol. In React any falsey value won't be rendered so shouldn't matter if we passed in undefined or what not.